src/rtc/h5_prep.py

```diff
@@ -300,20 +309,196 @@ def get_metadata_dict(product_id: str,
+        'RTC/metadata/fileFormatSpecification/dataMask/dataType':
+            ['mask_data_type', 'bytes',
+             'data type of the image mask'],
+        'RTC/metadata/fileFormatSpecification/dataMask/bitsPerSample':
+            ['mask_bits_per_sample', 8,
+             'Bytes per pixel'],
+        'RTC/metadata/fileFormatSpecification/dataMask//byteOrder':
+            ['mask_byte_order', sys.byteorder, # assuming that the byte order follows the native setting.
+             'Byte order of the data'],
+        'RTC/metadata/fileFormatSpecification/dataMask//bitValueRepresentation':
+            ['mask_bit_value_representation',
+             'no data = NaN, 0 = valid data, 1= ???, 2=???, 3=???', # TODO populate the string
+             'Description of the pixel values in the mask'],
+
+        # file format spec. for local inc. angle - for 2.4
+        'RTC/metadata/fileFormatSpecification/localIncidenceAngle/sampleType':  # 2.4
+            ['local_inc_sample_type', 'Angle', 'Sample type of the data'],
+        'RTC/metadata/fileFormatSpecification/localIncidenceAngle/dataFormat':
+            ['local_inc_data_format', cfg_in.groups.product_group.output_imagery_format,
+             'data format of the data'],
+        'RTC/metadata/fileFormatSpecification/localIncidenceAngle/dataType':
+            ['local_inc_data_type', 'Float',
+             'data type of the data'],
+        'RTC/metadata/fileFormatSpecification/localIncidenceAngle/bitsPerSample':
+            ['local_inc_bits_per_sample', 32,
+             'Bytes per pixel'],
+        'RTC/metadata/fileFormatSpecification/localIncidenceAngle/byteOrder':
+            ['local_inc_byte_order', sys.byteorder,
+             'Byte order of the data'],
+
+        # file format spec. for backscatter - for 3.1
+        'RTC/metadata/fileFormatSpecification/backScatter/measurementType':  # 3.1
+            ['measurement_type', 'Gamma0', 'Sample type of the data'],
+        'RTC/metadata/fileFormatSpecification/backScatter/expressionConvention':  # 3.1
+            ['measurement_expression_convention', 'Linear amplitude', #TODO Confirm
+             'Backscatter expression convention'],
+        # NOTE: polarization is provided in 'listOfPolarizations'
+        'RTC/metadata/fileFormatSpecification/backScatter/dataFormat':
+            ['measurement_data_format', cfg_in.groups.product_group.output_imagery_format,
+             'data format of the measurement'],
+        'RTC/metadata/fileFormatSpecification/backScatter/dataType':
+            ['measurement_data_type', 'Float',
+             'data type of the measurement'],
+        'RTC/metadata/fileFormatSpecification/backScatter/bitsPerSample':
+            ['measurement_bits_per_sample', 32,
+             'Bytes per sample in the measurement'],
+        'RTC/metadata/fileFormatSpecification/backScatter/byteOrder':
+            ['measurement_byte_order', sys.byteorder,
+             'Byte order of the measurement'],
```

Comment:
```suggestion
```
Thank you, @seongsujeong .  I think we can remove these lines.  These layers (RTC-S1 imagery and radar geometry layers) will be provided as Cloud-Optimized GeoTIFFs. All these parameters should be part of the GeoTIFF specification with the exception of the "sample type"/"measurement type", which will be saved in the band description.